src/Compilers/Server/VBCSCompiler/NamedPipeClientConnection.cs

```diff
@@ -229,6 +202,44 @@ internal NamedPipeClientConnection(ICompilerServerHost compilerServerHost, strin
             }
         }
 
+        protected override void ValidateBuildRequest(BuildRequest request)
```

Comment:
I was actually hoping to have a unit-test for the logic in this particular override rather than a unit-test that ensures that the method is called.